.github/workflows/test-spyre.yml

```diff
@@ -81,8 +81,11 @@ jobs:
         run: |
           uv venv .venv --system-site-packages
           source .venv/bin/activate
+
+          # Syncs both the runtime and dev deps, based on the lockfile contents
+          uv sync --frozen
+          # Builds and installs the vllm_spyre wheel into .venv
```

Comment:
`uv sync` will uninstall the installed wheel and instead point back to the local source, so the pip install needs to come last here

Reply:
👍🏻